Project.toml

```diff
@@ -34,6 +36,7 @@ ArrayInterface = "6, 7"
 CUDA = "5"
 DiffEqBase = "6"
 DiffEqCallbacks = "2 - 3.1, 3.8"
```

Comment:
can you also update the compat for `DiffEqCallbacks`, like the following ?
```julia
DiffEqCallbacks = "2 - 3.1, 3.8 - 3"
```
cause they released `v3.9`

Reply:
I think that this form should already accept versions up to 4.0 (excluded)

Reply:
ok